gwvolman/dataone_metadata.py

```diff
@@ -248,7 +248,7 @@ def create_minimum_eml(tale,
     description = tale.get('description', str())
     if description is not str():
         abstract = ET.SubElement(dataset, 'abstract')
-        ET.SubElement(abstract, 'para').text = strip_html_tags(description)
+        ET.SubElement(abstract, 'para').text = strip_html_tags(str(description))
```

Comment:
Per our conversation, I understand that you're using str() and strip_html_tags() to get a non-markdown/non-html version of the description. It would be helpful to have an example description that caused the problem that resulted in using the str() function.